x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/util/StringUtils.java

```diff
@@ -398,23 +398,50 @@ public static boolean isQualified(String indexWildcard) {
 
     public static boolean isInteger(String value) {
         for (char c : value.trim().toCharArray()) {
-            if (Character.isDigit(c) == false) {
+            if (isDigit(c) == false) {
                 return false;
             }
         }
         return true;
     }
 
+    private static boolean isLetter(char c) {
```

Comment:
Better to use Character.isLetter() since it's unicode aware.

Reply:
Discussed with @costin , checking a-zA-Z here is to make it consistent with lexer.